test/test_adjtrans.jl

```diff
@@ -0,0 +1,31 @@
+function test_adjtrans()
+  @testset ExtendedTestSet "Adjoint/Transpose/Conjugate" begin
+    A = rand(5,3) + im * rand(5,3)
+    opA = LinearOperator(A)
+
+    aopA = AdjointLinearOperator(opA)
+    copA = ConjugateLinearOperator(opA)
+    topA = TransposeLinearOperator(opA)
+
+    for (foo,fop) in [(adjoint, aopA),
+                      (conj, copA),
+                      (transpose, topA)]
+      @test foo(opA) === fop
+      @test Matrix(fop) == foo(A)
+      @test foo(fop) === opA
+
+      @test Matrix(-fop)    == foo(-A)
+      @test Matrix(2 * fop) == foo(2 * A)
```

Comment:
Better add a test with a complex number.

Reply:
Thanks, I've fixed the test to `x * foo(A)` as well.